osf/migrations/0001_initial.py

```diff
@@ -148,7 +148,7 @@ class Migration(migrations.Migration):
                 ('additional_metadata', osf.utils.datetime_aware_jsonfield.DateTimeAwareJSONField(blank=True, encoder=osf.utils.datetime_aware_jsonfield.DateTimeAwareJSONEncoder, null=True)),
             ],
             options={
-                'permissions': (('view_node', 'Can view node details'), ('read_node', 'Can read the node'), ('write_node', 'Can edit the node'), ('admin_node', 'Can manage the node')),
```

Comment:
this is only showing up as a duplication admin permission.

Reply:
ah, this is unrelated to the built-in migrations `view_` since mode name is `AbstractNode`.

however, I am not sure if it is a duplicate since I am not sure if "can view node details" is the same as "can read the node".

```python
permissions = (
    ('view_node', 'Can view node details'),
    ('read_node', 'Can read the node'),
    ('write_node', 'Can edit the node'),
    ('admin_node', 'Can manage the node'),
```

What issue did this one gave you?

Reply:
I don't think this is a duplicate permission.

```SQL
SELECT * FROM auth_permission WHERE codename='view_node';
```
| id | name | content\_type\_id | codename |
| :--- | :--- | :--- | :--- |
| 73 | Can view node details | 7 | view\_node |
| 515 | Can view node | 14 | view\_node |

The `pk=515` `view_node` is the new built-in permission from Django for `Node`
The `pk=73` `view_node` is the one we defined for `AbstractNode`

However, `permission_required = 'osf.view_node'` seems to honor any of them when provided as the permission, whether using

```python
permission = Permission.objects.filter(
    content_type_id=ContentType.objects.get_for_model(Node).id,
    codename='view_node').first()
)
```

or 

```python
permission = Permission.objects.filter(
    content_type_id=ContentType.objects.get_for_model(AbstractNode).id
    codename='view_node'
).first()
```

I am not 100% sure what the best fix is but removing `view_node` from `AbstractNode` seems to work. And if we end up with this, we need a migration instead of modifying the `0001_initial`.